docs/demand/historical/texas.rst

```diff
@@ -0,0 +1,5 @@
+Texas
+#####
+ERCOT publishes historical hourly load data for its eight control areas. The data
```

Comment:
We should replace `control areas` by `weather zones`, according to their official names, details check [here](http://www.ercot.com/news/mediakit/maps).

Reply:
Done